.github/workflows/cd.yml

```diff
@@ -145,19 +172,22 @@ jobs:
     needs:
       - setup
     with:
-      branch: ${{ github.event.inputs.branch }}
-      run-playwright: ${{ github.event.inputs.run-playwright == true }}
+      branch: ${{ inputs.branch }}
+      go-version: ${{ inputs.go-version }}
+      node-version: ${{ inputs.node-version }}
+      golangci-lint-version: ${{ inputs.golangci-lint-version }}
+      run-playwright: ${{ inputs.run-playwright == true }}
       run-playwright-with-grafana-dependency: ${{ inputs.run-playwright-with-grafana-dependency }}
       run-playwright-with-skip-grafana-dev-image: ${{ inputs.run-playwright-with-skip-grafana-dev-image }}
       run-playwright-with-version-resolver-type: ${{ inputs.run-playwright-with-version-resolver-type }}
-      upload-playwright-artifacts: ${{ github.event.inputs.upload-playwright-artifacts == true }}
-      run-trufflehog: ${{ github.event.inputs.run-trufflehog == true }}
+      upload-playwright-artifacts: ${{ inputs.upload-playwright-artifacts == true }}
```

Comment:
Is changing `github.event.inputs` to just `inputs`(stripping out the `github.event`) related to 

> The workflow will also receive the inputs in the github.event.inputs context. The information in the inputs context and github.event.inputs context is identical except that the inputs context preserves Boolean values as Booleans instead of converting them to strings. The choice type resolves to a string and is a single selectable option.
?

Reply:
Ah, I didn't know that. I only renamed them because I thought it was too verbose to have `github.event`... and `inputs` was working just fine :sweat_smile: . Great catch! I can refactor to account for that

Reply:
Done!